sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -917,7 +931,7 @@ def arrow_schema_to_bq_schema(arrow_schema: pyarrow.Schema) -> List[SchemaField]
             ,created_timestamp
         {% endif %}
     )
-){% if loop.last %}{% else %}, {% endif %}
```

Comment:
haven't taken a close look, but seems like you removed this line even though there's a loop. Was that intended?